File: Back/src/ControleCustos.Domain/Fornecedor.cs
using System.Collections.Generic;
using ControleCustos.Domain.Identity;
using ProEventos.Domain.Enum;

namespace ControleCustos.Domain
{
    public class Fornecedor
    {
        public int FornecedorId { get; set; }
        public string Nome { get; set; }
        public int TipoFornecimento { get; set; }
        public int UserId { get; set; }

        public User User { get; set; }
        public IEnumerable<Conta> Contas { get; set; }
    }
}

File: Front/ControleCustos-APP/src/app/models/identity/Fornecedor.ts
import { Conta } from "./Conta"
import { User } from "./User"

export class Fornecedor {
   fornecedorId: number;
   nome: string;
   tipoFornecimento: string;
   userId: number;
   ativo?: boolean;

   user: User;
   contas: Conta[];
}
